controllers/controllers.errors.js

```diff
@@ -0,0 +1,6 @@
+const handle404Paths = (req, res) => {
```

Comment:
I suggest extracting this out into its own file in the root directory 'error-handling.js' as these are technically not controllers. These are error handling middleware functions.